testing/dockertest/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM debian:latest
```

Comment:
Agreed, when beginning these tests Alpine gave me trouble with downloading certain dependencies but this may have been an issue I ran into with Docker.